service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -127,6 +127,19 @@
     private final List<Element<Direction>> respondentDirections;
     private final List<Element<Direction>> respondentDirectionsCustom;
 
+    @JsonIgnore
+    public List<Element<Direction>> getDirectionsToComplyWith() {
+        if (getServedCaseManagementOrders().isEmpty() && standardDirectionOrder == null) {
+            return emptyList();
+        }
+
+        if (getServedCaseManagementOrders().isEmpty()) {
+            return standardDirectionOrder.getDirections();
+        }
+
+        return servedCaseManagementOrders.get(0).getValue().getDirections();
```

Comment:
this could be return unwrapElements(servedCaseManagementOrders).get(0).getDirections(); but is probably fine as is if it's only used the once in this method.

Reply:
I think this one is fine as it is.